fizzbuzz/COBOL/nathaniel_priestley/run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+echo "Begin shell script execution."
+echo $1 | ./FizzBuzz-exe
```

Comment:
you could put this compile step directly into this shell script too to avoid having this exe file